.changeset/six-pianos-nail.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Whats with the filename? is that just a random string changeset generates?

Reply:
yup, that's what it does to make sure that it doesn't collide with changes in other PRs.